src/amo/components/SearchForm/styles.scss

```diff
@@ -0,0 +1,80 @@
+@import "~core/css/inc/mixins";
+@import "~amo/css/inc/vars";
+@import "~ui/css/vars";
+
+$item-padding-horizontal: 23px;
+$item-padding: 5px $item-padding-horizontal;
+$arrow-width: 18px;
+$addon-icon-margin-end: 12px;
+
+.SearchForm-form {
+  background: $header-search-color;
+  border: 0;
+}
+
+.SearchInput-input,
+.SearchForm-form {
+  color: $base-color;
+  font-size: $font-size-m-smaller;
+  white-space: nowrap;
+}
+
+.SearchForm-suggestions-list {
+  background-color: $white;
+  color: $black;
+  margin: 0;
+  padding: 0;
+}
+
+.SearchForm-suggestions-item {
+  @include text-align-start();
+
+  cursor: pointer;
+  outline: none;
+  padding: $item-padding;
+  width: 100%;
+
+  .Icon-arrow-big-blue {
+    display: none;
+    float: right;
+
+    [dir=rtl] & {
+      float: left;
+    }
+  }
+}
+
+.SearchForm-suggestions-item:hover,
+.SearchForm-suggestions-item:active,
+.SearchForm-suggestions-item:focus,
+.SearchForm-suggestions-item--highlighted {
+  color: $link-color;
+  white-space: normal;
+
+  .Icon-arrow-big-blue {
+    display: inline-block;
+  }
+}
+
+.Suggestion {
+  height: 20px;
```

Comment:
When suggestions arrive and they replace the `LoadingText` placeholders, I see everything expand by about 3 or 4 pixels. Did you look into why that's happening? It might just need some `.Suggestion .LoadingText {...}` styles.